tajo-plan/src/main/java/org/apache/tajo/plan/LogicalPlanner.java

```diff
@@ -577,7 +577,7 @@ public static void verifyProjectedFields(QueryBlock block, Projectable projectab
   public static void prohibitNestedRecordProjection(Projectable projectable)
       throws TajoException {
     for (Target t : projectable.getTargets()) {
-      if (t.getEvalTree().getValueType().getType() == TajoDataTypes.Type.RECORD) {
+      if (t.getEvalTree().getValueType().isStruct()) {
```

Comment:
Currently, record is right. I need to make them consistent. According to my future plan, struct would be better. That's why they are mixed.